fuzzers/forkserver/forkserver_simple/src/main.rs

```diff
@@ -205,7 +221,35 @@ pub fn main() {
         HavocScheduledMutator::with_max_stack_pow(havoc_mutations().merge(tokens_mutations()), 6);
     let mut stages = tuple_list!(StdMutationalStage::new(mutator));
 
-    fuzzer
-        .fuzz_loop(&mut stages, &mut executor, &mut state, &mut mgr)
-        .expect("Error in the fuzzing loop");
+    if !debug_child {
+        fuzzer
+            .fuzz_loop(&mut stages, &mut executor, &mut state, &mut mgr)
+            .expect("Error in the fuzzing loop");
+    } else {
+        loop {
+            fuzzer
+                .fuzz_one(&mut stages, &mut executor, &mut state, &mut mgr)
+                .expect("Error in fuzz_one");
+
+            println!(
```

Comment:
this is for debug?

Reply:
This is showcasing that we are capturing the stdout/stderr.